get_deps.py

```diff
@@ -33,7 +33,10 @@ def main():
 		if dll_path.lower().startswith('/c/windows') or dll_path.lower().startswith('c:/windows'):
 			print('Skipping system DLL %s' % dll_path)
 			continue
+
 		print('Copying %s...' % dll_path)
+		# Python wants Windows-style paths, not Unix-style
+		dll_path = subprocess.check_output(['cygpath', '-w', dll_path]).strip()

```

Comment:
Shouldn't this be done before:

```
if dll_path.lower().startswith('/c/windows') or dll_path.lower().startswith('c:/windows'): 
```

so that it can be simplified?

---

What happens if `cygpath` binary does not exist?